civil/civil.go

```diff
@@ -86,6 +86,17 @@ func (d Date) AddDays(n int) Date {
 	return DateOf(d.In(time.UTC).AddDate(0, 0, n))
 }
 
+// AddDate returns the time corresponding to adding the given number of years,
+// months, and days to t. For example, AddDate(-1, 2, 3) applied to January 1,
+// 2011 returns March 4, 2010.
+//
+// AddDate normalizes its result in the same way that Date does, so, for
+// example, adding one month to October 31 yields December 1, the normalized
+// form for November 31.
+func (d Date) AddDate(years, months, days int) Date {
```

Comment:
For my own edification: if one of these values is `0`, that's ok, because it could just be "add one year and one day" for `AddDate(1, 0, 1)`. Right?